CMakeLists.txt

```diff
@@ -20,7 +20,6 @@ option(BUILD_OSTREE "Set to ON to compile with ostree support" ON)
 option(ALLOW_MANUAL_ROLLBACK "Set to ON to build with support of manual rollbacks" OFF)
 option(BUILD_AKLITE_OFFLINE "Set to ON to build cli command for an offline update" OFF)
 option(BUILD_AKLITE_WITH_NERDCTL "Set to ON to build with support of nerdctl/containerd" OFF)
-option(BUILD_AKLITE_APPS "Set to ON to build cli command for Apps management" ON)
```

Comment:
Seems like this is going to cause changes to https://docs.foundries.io/latest/user-guide/container-preloading/container-preloading.html ? what's the plan for that? ie can this break someone using container-preloading?

Reply:
No, the given change doesn't cause changes to the preloading.

Reply:
The commit 1e05cd3d4e5da5c9e98fe554fb48b02fc204090d only removes aklite apps. It does not turn on composectl usage.

Reply:
But that document refers to the `aklite-apps` tool which is going away. Is it simply a matter of removing the text or do we have people that use this tool? Or is this a "release notes" thing where people upgrading will have to change something?

Reply:
The corresponding meta-lmp change creates a symbolic link `aklite-apps` that refers to `composectl` https://github.com/foundriesio/meta-lmp/pull/1369/commits/4243e880cda2c03c5b71a373527fcf450793b1c3#diff-5d7444c561f7fd4d3c4b9d87080fecb23114011e6530898be51380df7fd2b2c4R24.

Reply:
ah. i assumed the first release of this would have it optional. We are going to require the new utlitlity (if you use aklite-apps). I guess the aklite-apps part of this PR could really be its own unique PR and is almost orthogonal to the introduction of composeapp as an app engine.

Minor thing but maybe something to consider in future release notes. I'm guessing the size of `composectl` is a bit bigger than the old `aklite-apps`. We might want to note that or at least know it ourselves.

Reply:
Removing `skopeo` compensate diff between `composectl` and `aklite-apps` sizes.
Sure, we can add info about approximate size of `composectl`.

Reply:
it helps the *total* size. I was thinking about the cost of the OTA

Reply:
ah, ok.

FYI (arm64, rpi4)
```
du -sh /bin/composectl
9.0M	/bin/composectl
```